generators/client/templates/src/main/webapp/app/blocks/config/_compile.config.js

```diff
@@ -5,13 +5,15 @@
         .module('<%=angularAppName%>')
         .config(compileServiceConfig);
 
-    compileServiceConfig.$inject = ['$compileProvider', 'ENV'];
+    compileServiceConfig.$inject = ['$compileProvider'];
 
-    function compileServiceConfig($compileProvider, ENV) {
+    function compileServiceConfig($compileProvider) {
         // disable debug data on prod profile to improve performance
-        if(ENV === 'prod'){
-            $compileProvider.debugInfoEnabled(false);
-        }
+        
+        // TODO: Howto fetch the dynamic profile in this config.... can this be done elsewhere ?
```

Comment:
Tried the ```injector.get``` but didn't work. Don't think you're supposed to inject services during app config. [Angular docs](https://docs.angularjs.org/guide/module) say : 

_Configuration blocks - get executed during the provider registrations and configuration phase. Only providers and constants can be injected into configuration blocks. This is to prevent accidental instantiation of services before they have been fully configured._